README.md

```diff
@@ -26,15 +27,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+data-ng-click is able to be validated by an html validator.
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+A - ng-app should go in the location you'd like your angular app to be recognized to start. since <html> is the first tag, it makes sense to put it there as a best practice but also to support other coders being able to find it.
```

Comment:
It's not because it's the first tag. It's because we want to be able to reference our Angular app throughout all of our HTML.